lib/song.rb

```diff
@@ -0,0 +1,60 @@
+class Song
+  extend Concerns::Findable
+
+  @@all = []
+
+  def initialize(name, artist = nil, genre = nil)
+    @name = name
+    @artist = artist
```

Comment:
Consider using the setter method **artist=** since it already implements the *add_song* functionality. **DRY**.